SpaceShooterGame/SpaceShooterGame/GameScene.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  GameScene.swift
+//  SpaceShooterGame
+//
+//  Created by Tristan Garzon on 2018-03-07.
+//  Copyright © 2018 Tristan Garzon. All rights reserved.
+//
+
+
+//Notes: Timer, Enemies, Firing
+
+import SpriteKit
+import GameplayKit
+
+class GameScene: SKScene, SKPhysicsContactDelegate {
```

Comment:
It conforms to SKPhysicsContactDelegate but doesn't implement any of the delegate methods? All the methods must be optional but why do you need it to be the delegate then?